source/api/discovery/0.5/index.md

```diff
@@ -446,27 +458,28 @@ The identifier of the Collection Page.
 
 Ordered Collection Pages _MUST_ have an `id` property. The value _MUST_ be a string and it _MUST_ be an HTTP(S) URI. The JSON representation of the Ordered Collection Page _MUST_ be available at the URI.
 
-```
+```json-doc
 { "id": "https://example.org/activity/page-0" }
 ```
 
 ##### type
+{: #type-ordered-collection-page}
 
 The class of the Ordered Collection Page.
 
-Ordered Collections _MUST_ have a `type` property.  The value _MUST_ be `OrderedCollectionPage`.
+Ordered Collections _MUST_ have a `type` property. The value _MUST_ be `OrderedCollectionPage`.
 
-```
+```json-doc
 { "type": "OrderedCollectionPage" }
 ```
 
 ##### partOf
 
-The Ordered Collection that this Page is part of.
+The Ordered Collection of which this Page is a part.
```

Comment:
I've managed to keep us from ending sentences w/ prepositions up to now...

Reply:
:)

Reply:
My sentence was influenced by the name of the property, I expect.  :+1: to getting rid of dangling prepositions!